src/parser/jobs/sam/modules/Meikyo.js

```diff
@@ -57,39 +85,81 @@ export default class Meikyo extends Module {
 
 	_onComplete() {
 
-		if (this._badMeikyoCasts > 0) {
-			this.suggestions.add(new TieredSuggestion({
-				icon: ACTIONS.MEIKYO_SHISUI.icon,
-				content: <Trans id= "sam.meikyo.suggestions.badmeikyo.content">
+		//invuln check time for drift
+
+		const INVULN = (this.invuln.getInvulnerableUptime()/1000)
+		//const FORGIVEN_DRIFT = Math.floor(INVULN / MEIKYO_COOLDOWN) //this will calculate the amount of full recasts during a fight where a person could not use the skill.
+		const FIGHT_DURATION = ((this.parser.fightDuration/1000) - INVULN)
+		const EXPECTED_MEIKYO = Math.floor(FIGHT_DURATION/ MEIKYO_COOLDOWN)
```

Comment:
likely a total non-issue; but in a fight precisely 10m long, a 1m CD can be used 11 times, rather than just 10 - so this might be undershooting a tad.

Reply:
Is this a on the minute mark problem or just always + 1 because things start off cooldown?